package.json

```diff
@@ -53,6 +53,7 @@
 		"react-native-fast-image": "^8.6.3",
 		"react-native-gesture-handler": "~2.14.0",
 		"react-native-linear-gradient": "^2.8.3",
+		"react-native-onboarding-swiper": "^1.2.0",
```

Comment:
This library does not look very good. It has no updates fro 2 years now and has only 10k downloads per year. I understand that it works now, but RN ecosystem is in state of great shift (https://www.youtube.com/watch?v=BdPkVl4Y8TQ) so unmaintained libraries like this are a source of headache in future